torchdata/dataloader2/communication/iter.py

```diff
@@ -263,6 +263,7 @@ def pause(self):
                 if NonBlocking.not_available_hook is not None:
                     NonBlocking.not_available_hook()
 
+    @final
```

Comment:
nit: Don't think this is needed either? Since no one should be inheriting from this class (maybe developers/ourselves, but we are doing that)?

I wouldn't mind if you decide to keep this

Reply:
In the end I decided to only keep the `@final` of the datapipes.